src/qt/walletmodel.cpp

```diff
@@ -514,6 +514,13 @@ bool WalletModel::bumpFee(uint256 hash, uint256& new_hash)
     questionString.append("</td><td>");
     questionString.append(BitcoinUnits::formatHtmlWithUnit(getOptionsModel()->getDisplayUnit(), new_fee));
     questionString.append("</td></tr></table>");
+
+    //Check if "Coin Control Features" options is enabled to display warning in "Confirm fee bump" Window
+    if (getOptionsModel()->getCoinControlFeatures() == true) {
+    questionString.append("<br><br>");
+    questionString.append(tr("Warning: This may pay the additional fee by reducing change outputs or adding inputs, when necessary. It may add a new change output if one does not already exist. These may potentially leak privacy."));
+    }
```

Comment:
you can run ` clang-format -i src/qt/walletmodel.cpp --lines=518:522` to format your code automatically
```diff
     if (getOptionsModel()->getCoinControlFeatures() == true) {
-    questionString.append("<br><br>");
-    questionString.append(tr("Warning: This may pay the additional fee by reducing change outputs or adding inputs, when necessary. It may add a new change output if one does not already exist. These may potentially leak privacy."));
+        questionString.append("<br><br>");
+        questionString.append(tr("Warning: This may pay the additional fee by reducing change outputs or adding inputs, when necessary. It may add a new change output if one does not already exist. These may potentially leak privacy."));
     }
```

Reply:
Thanks